django_recaptcha/static/django_recaptcha/js/widget_v2_checkbox.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+
+  function main() {
+    for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+      const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
+      if (!widgetUuid) {
+        console.warn("reCAPTCHA widget with missing UUID");
+        break;
+      }
+      console.log(`found reCAPTCHA widget with UUID '${widgetUuid}'`);
```

Comment:
This strikes me as too verbose, we shouldn’t be adding extra logging to projects that use those widgets.